source/query-with-natural-language/prompt-natural-language-query.txt

```diff
@@ -9,6 +9,99 @@ Prompt your First Natural Language Query
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 2
+   :depth: 1
 
-Prompt your First Natural Language Query
\ No newline at end of file
+You can use MongoDB Compass to generate filter documents using the
+natural language querying feature. This feature automatically generates 
+filter documents based on prompts you provide to the MongoDB AI engine. 
+Natural language querying serves as a helpful starting point and assists
+you in writing MongoDB queries.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-natural-language-query.rst
+
+The MongoDB AI engine can return both filter documents and 
+aggregations. If your prompt results in an aggregation, a popup
+appears directing you to the :guilabel:`Aggregations` tab.
+
+This page uses the :ref:`sample_mflix.movies <sample-mflix>` 
+collection from the Atlas sample dataset.
+
+Before you Begin
+----------------
+
+You must :ref:`enable the natural language querying feature<enable-natural-language-querying>`.
+
+Steps
+-----
+
+.. procedure::
+   :style: connected
+
+   .. step:: Select the :guilabel:`Documents` tab
+
+   .. step:: Click the :guilabel:`Ask AI` button
+
+      The :guilabel:`Natural Language Query Bar` appears.
+
+   .. step:: Type a natural language prompt for your collection
+
+      Enter a prompt in the :guilabel:`Natural Language Query Bar`.
+      For example: ``Which movies were released in 2000?``.
```

Comment:
(suggestion)
You could flip the content a little. Something like:
.. step:: Type a question for your collection
   - Type a natural language prompt int the the query bar. 
      For example: ... (fine to omit the period at the end.)
   - Press enter or click Generate.
   A filter document populates the Filter bar.

Reply:
Updated.